src/test/stepFunctions/commands/visualizeStateMachine.test.ts

```diff
@@ -0,0 +1,464 @@
+/*!
+ * Copyright 2018-2019 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+import * as assert from 'assert'
+import * as sinon from 'sinon'
+import * as vscode from 'vscode'
+import { Disposable } from 'vscode-languageclient'
+import { AslVisualizationManager } from '../../../../src/stepFunctions/commands/visualizeStateMachine/aslVisualizationManager'
+import { AslVisualization } from '../../../../src/stepFunctions/commands/visualizeStateMachine/aslVisualization'
+
+import { ext } from '../../../shared/extensionGlobals'
+import { StateMachineGraphCache } from '../../../stepFunctions/utils'
+
+// Top level defintions
+let aslVisualizationManager: AslVisualizationManager
+const sandbox = sinon.createSandbox()
```

Comment:
This should probably go into the `before` block to prevent any unexpected side effects